project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=1.0.0-RC2
```

Comment:
sbt's launcher (as installed by brew or others) is intended to be forward-compatible with sbt.version's defined in build.properties, fyi.